indra/llui/llfloater.cpp

```diff
@@ -659,6 +659,8 @@ void LLFloater::openFloater(const LLSD& key)
 
 	LLViewerEventRecorder::instance().logVisibilityChange( getPathname(), getName(), true,"floater"); // Last param is event subtype or empty string
 
+	LLMenuGL::sMenuContainer->hideMenus();
```

Comment:
Scenario:
Open a 'Build->uploads menu'
Detach menu (two lines at the top of 'uploads' sub menu)
Expected: for main menu not to close and allow user to detach more menus

Reply:
If any floater popup is going to close menu, it will be imposible to navigate menus while receiving script questions and chat messages (with closed chat).

Reply:
> Open a 'Build->uploads menu'

Which menu item do you mean?
![image](https://github.com/secondlife/viewer/assets/124201357/12ae2121-dbc6-4083-a9ac-2f92eac64514)

Reply:
> Detach menu (two lines at the top of 'uploads' sub menu)

Screenshot please